modules/core/src/test/java/org/apache/ignite/hlc/HybridClockTestUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.hlc;
+
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import java.time.Clock;
+import java.time.Instant;
+import org.mockito.MockedStatic;
+
+/**
+ * Utils for a Hybrid Logical Clock testing.
+ */
+public class HybridClockTestUtils {
+    /**
+     * Creates a mocked system clock.
+     *
+     * @param expected Expected value which returns by system clock.
+     * @return The mocked clock.
+     */
+    public static MockedStatic<Clock> mockToEpochMilli(long expected) {
```

Comment:
Here is the citation from the internet on statick mocks:
"Generally speaking, some might say that when writing clean code, we shouldn't need to mock static classes. This could typically hint at a design issue or code smell in the application.

Why? First, a class depending on a static method has tight coupling, and second, it nearly always leads to code that is difficult to test. Ideally, a class should not be responsible for obtaining its dependencies, and if possible, they should be externally injected.

So, it's always worth investigating if we can refactor our code to make it more testable."
I suggest to take it into a consideration.